templates/hound.yml

```diff
@@ -0,0 +1,17 @@
+# See https://houndci.com/configuration for help.
+coffeescript:
+  # config_file: .coffeescript-style.json
+  enabled: false
```

Comment:
How about having them all be enabled by default, and it's up to the project wether they are using CS or JS (since Hound will review based on file extensions)? Possible that some projects might have a mix of CS and JS for whatever reasons.